person_capture/gui_app.py

```
@@ -4592,32 +4651,71 @@ def save_hit(c, idx):
-                            # enqueue a contiguous copy; slices are views into `frame`
-                            buf = np.ascontiguousarray(crop_img2)
-                            if not buf.flags.owndata:
-                                buf = buf.copy()
-                            save_q.put_nowait((crop_img_path, buf, row))
+                            if hdr_primary_fullres:
+                                save_q.put_nowait({
+                                    "type": "hdr_sdr",
+                                    "path": crop_img_path,
+                                    "row": row,
+                                    "frame_idx": int(idx),
+                                    "frame_pts_sec": frame_pts_sec,
```

Ruff (0.15.11)

[warning] Function definition does not bind loop variable `frame_pts_sec`

(B023)